otx/algorithms/segmentation/adapters/mmseg/models/heads/custom_fcn_head.py

```diff
@@ -4,30 +4,138 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+from mmcv.runner import force_fp32
 from mmseg.models.builder import HEADS
 from mmseg.models.decode_heads.fcn_head import FCNHead
+from mmseg.models.losses import accuracy
+from mmseg.ops import resize
+from torch import nn
 
-from .mixin import (
-    AggregatorMixin,
-    MixLossMixin,
-    PixelWeightsMixin2,
-    SegmentOutNormMixin,
+from otx.algorithms.segmentation.adapters.mmseg.models.utils import IterativeAggregator
+from otx.algorithms.segmentation.adapters.mmseg.utils import (
+    get_valid_label_mask_per_batch,
 )
 
 
 @HEADS.register_module()
-class CustomFCNHead(
-    SegmentOutNormMixin, AggregatorMixin, MixLossMixin, PixelWeightsMixin2, FCNHead
-):  # pylint: disable=too-many-ancestors
+class CustomFCNHead(FCNHead):  # pylint: disable=too-many-ancestors
     """Custom Fully Convolution Networks for Semantic Segmentation."""
 
-    def __init__(self, *args, **kwargs):
+    def __init__(
+        self,
+        enable_aggregator=False,
```

Comment:
How about adding type hint for this class?

Reply:
Thanks for comments. I added.